ui/src/components/home/HeroSection.js

```diff
@@ -32,7 +32,7 @@ const HeroSection = ({ anchorTarget }) => {
         as="video"
         ref={vidRef}
         w="full"
-        h={['92vh', '96vh']}
+        h={['565px', '96vh']}
```

Comment:
why use `px`? we usually use vw, vh, em, because it's responsive to the screen size. if it's px, it wont change responsively for mobile and desktop

Reply:
Okay noted, thanks for the guidance. Because sometimes UI/UX team gives specific widths and font sizes so I wasn't sure whether do use responsive CSS units or not, and those fixed CSS units seemed alright when I change the screen size